demo/src/main/java/com/eis/geoCalendar/demo/Behaviour/wrapper/NetworkEventMapBehaviourWrapper.java

```diff
@@ -0,0 +1,233 @@
+package com.eis.geoCalendar.demo.Behaviour.wrapper;
+
+import android.graphics.Bitmap;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.FragmentManager;
+
+import com.eis.geoCalendar.demo.Behaviour.LocationRetriever;
+import com.eis.geoCalendar.demo.Behaviour.MapBehaviour;
+import com.eis.geoCalendar.demo.Behaviour.NetworkEventMapBehaviour;
+import com.eis.geoCalendar.demo.Behaviour.NetworkMapBehaviour;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventCreatedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventRemovedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventTriggeredListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnMapInitializedListener;
+import com.eis.geoCalendar.demo.Bottomsheet.BottomSheetBehaviour;
+import com.eis.geoCalendar.demo.Dialogs.AbstractAddEventDialog;
+import com.eis.geoCalendar.demo.Dialogs.AbstractRemoveEventDialog;
+import com.eis.geoCalendar.demo.Localization.GoToGoogleMapsNavigator;
+import com.eis.geoCalendar.events.Event;
+import com.eis.geoCalendar.network.NetworkEventUser;
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.List;
+
+/**
+ * Class that wraps public methods for {@link com.eis.geoCalendar.demo.Behaviour.NetworkEventMapBehaviour}
+ * following interface {@link com.eis.geoCalendar.demo.Behaviour.MapBehaviour} and
+ * {@link com.eis.geoCalendar.demo.Behaviour.NetworkMapBehaviour}
+ * <p>
+ * If needed, the class {@link com.eis.geoCalendar.demo.Behaviour.NetworkEventMapBehaviour} can be set
+ * to package-private and this wrapper can be used outside of the package
+ *
+ * @author Turcato
+ */
+public class NetworkEventMapBehaviourWrapper implements MapBehaviour<Event<String>>, NetworkMapBehaviour {
```

Comment:
Same as EventMapBehaviourWrapper: if you want, you should add {@link className}.